apps/scan/frontend/src/screens/poll_worker_screen.tsx

```diff
@@ -148,13 +148,20 @@ function OpenPollsPromptScreen({
           <Button
             variant="primary"
             onPress={onConfirm}
-            disabled={!shouldAllowTogglingPolls(printerSummary, usbDriveStatus)}
+            disabled={
+              !shouldAllowTogglingPolls(
+                printerSummary,
+                mustInsertUsbDriveToContinue
+              )
+            }
           >
             Open Polls
           </Button>
         </P>
         <PrinterAlertText printerSummary={printerSummary} />
-        <UsbDriveAlertText usbDriveStatus={usbDriveStatus} />
+        <UsbDriveAlertText
+          mustInsertUsbDriveToContinue={mustInsertUsbDriveToContinue}
+        />
```

Comment:
nit: Rather than having `UsbDriveAlertText` internally render `null` when `mustInsertUsbDriveToContinue` is `false`, I think the code may read clearer if we pull that check out of the component, i.e.

```tsx
{mustInsertUsbDriveToContinue && <UsbDriveAlertText />}
```

Reply:
addressed in follow-up commit